devicePath: /dev/evarobotSonar
alwaysOn: true
frequency: 10.0
field_of_view: 0.7
minRange: 0.0
maxRange: 5.0
maxFreq: 10.0
minFreq: 0.2
element: 1
sonarPins: [0, 1, 2, 3, 4, 5, 6]
